tests/integration/test_content_manifest.py

```diff
@@ -19,3 +19,43 @@ def test_invalid_content_manifest_request(test_env):
         client.fetch_content_manifest(request_id=0)
     assert e.value.response.status_code == 404
     assert e.value.response.json() == {"error": "The requested resource was not found"}
+
+
+def test_valid_content_manifest_request(test_env, default_request):
+    """
+    Send a valid content-manifest request to the Cachito API.
+
+    Checks:
+    * Check that the response code is 200
+    * Check that the returned content contains a URL
+    pointing to a JSON schema for the content manifest file
+    * Check that the JSON schema is the same as in atomic-reactor repository
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+
+    initial_response = default_request.initial_response
+    content_manifest_response = client.fetch_content_manifest(initial_response.data["id"])
+    assert content_manifest_response.status == 200
+
+    response_data = content_manifest_response.data
+    assert_content_manifest_response(test_env, response_data)
+
+    icm_spec = response_data["metadata"]["icm_spec"]
+    assert_content_manifest_schema(test_env, icm_spec)
+
+
+def assert_content_manifest_response(test_env, response_data):
+    """Check the response data from Cachito content manifest."""
+    json_schema_url = test_env["content_manifest"]["expected_icm_spec"]
+    expected_response_data = {
+        "metadata": {"icm_version": 1, "icm_spec": json_schema_url, "image_layer_index": -1}
```

Comment:
It could be better to also have the values of `icm_version` and `image_layer_index` in test_env.